modes/event_types.md

```diff
@@ -21,7 +23,7 @@ As with all MDS definitions, they should be described in a way that maximizes th
 | `missing`             | Missing |
 | `off_hours`           | Off hours - end of service |
 | `on_hours`            | On hours - start of service |
-| `passenger_cancellation` | Passenger cancelled a trip |
+| `customer_cancellation` | Customer cancelled a trip |
```

Comment:
the notion of passenger does not fit the deliveries. Therefore, it has been replaced by the notion of customer